src/util/windows.jl

```diff
@@ -87,8 +87,9 @@ end
 
 @inline function Base.next(it::EachWindowIterator, state::Integer)
     i = state
-    window = sub(it.data, i:i + it.width - 1)
-    return window, i + it.step
+    region = i:i + it.width - 1
```

Comment:
If the window size is fixed, returning the end position is not needed, which is also compatible with our each-kmer iterator.